nglview/adaptor.py

```diff
@@ -348,4 +349,37 @@ def get_structure_string(self):
         self.mol.write(fname)
         pdb_string = os.fdopen(fd).read()
         # os.close( fd )
-        return pdb_string
\ No newline at end of file
+        return pdb_string
+
+
+@register_backend('asetraj')
```

Comment:
`asetraj` should be `ase` since this code will check the package name.

You can test that by

```
from ase.io.trajectory import Trajectory
traj = Trajectory('example.traj')
view = nglview.NGLWidget()
view.add_trajectory(traj) 
# so you don't need to create ase_traj = ASETrajectory(traj)
view
```